RdkServicesTest/Tests/DeviceDiagnosticsTest.cpp

```diff
@@ -0,0 +1,85 @@
+/**
+* If not stated otherwise in this file or this component's LICENSE
+* file the following copyright and licenses apply:
+*
+* Copyright 2020 RDK Management
+*
+* Licensed under the Apache License, Version 2.0 (the "License");
+* you may not use this file except in compliance with the License.
+* You may obtain a copy of the License at
+*
+* http://www.apache.org/licenses/LICENSE-2.0
+*
+* Unless required by applicable law or agreed to in writing, software
+* distributed under the License is distributed on an "AS IS" BASIS,
+* WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+* See the License for the specific language governing permissions and
+* limitations under the License.
+**/
+
+#include "gtest/gtest.h"
+
+#include "DeviceDiagnostics.h"
+
+#include "Source/WorkerPoolImplementation.h"
+#include "Source/Config.h"
+#include "Source/Service.h"
+
+namespace RdkServicesTest {
+
+TEST(DeviceDiagnosticsTest, test) {
```

Comment:
Can we split into different tests instead of a single test.

Reply:
This is small test (only 2 requests) I think we can split larger tests like `SecurityAgentTest.cpp`.